tests/unit/curated_recommendations/test_rankers.py

```diff
@@ -356,6 +360,36 @@ def test_boost_preferred_topic_no_reorder(self):
         assert recs == not_reordered_recs
 
 
+class TestIsSectionRecentlyFollowed:
+    """Unit tests for is_section_recently_followed"""
+
+    @freezegun.freeze_time("2025-03-20 12:00:00", tick=True, tz_offset=0)
```

Comment:
Why set `tick=True`? I think you want [tick](https://github.com/spulec/freezegun?tab=readme-ov-file#tick-argument) to be false for all these tests, which is the default. When set to true, the time keeps ticking during the test, which you sometimes need if there's a sleep call.
```suggestion
    @freezegun.freeze_time("2025-03-20 12:00:00", tz_offset=0)
```